examples/react/start-workos/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "tanstack-start-example-workos",
+  "version": "1.0.0",
+  "private": true,
+  "type": "module",
+  "main": "index.js",
+  "scripts": {
+    "dev": "vinxi dev",
+    "build": "vinxi build",
+    "start": "vinxi start",
+    "format": "prettier --write ."
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "description": "",
+  "dependencies": {
+    "@radix-ui/themes": "^3.2.1",
+    "@tanstack/react-router": "^1.115.2",
+    "@tanstack/react-router-devtools": "workspace:^",
```

Comment:
we use absolute version numbers in example.

Reply:
Thanks! I went ahead and updated.